plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/params/BuildImageParams.java

```diff
@@ -182,6 +188,9 @@ public BuildImageParams withFiles(@NotNull File... files) {
      *         if {@code files} is null
      */
     public BuildImageParams addFiles(@NotNull File... files) {
+        if (remote != null) {
```

Comment:
Not necessarily. It is still possible to do this: `BuildImageParams.create(<remote>).addFiles(<a file>)`. And `NPE` says nothing about root of error.